Cargo.toml

```diff
@@ -28,6 +28,7 @@ mysql_common = "0.22"
 byteorder = "1"
 chrono = "0.4"
 time = "0.2.25"
+rustls = "0.20.0"
```

Comment:
I wonder if we should make TLS support optional to save people some compile time. Basically an on-by-default feature that we gate all the TLS stuff behind.

Reply:
Done.